tests/onnx/quantization/test_weights_compression.py

```diff
@@ -284,4 +284,4 @@ def test_matmulnbits():
     output21 = sess21.run(None, {"input": dummy_input})[0]
     output19 = sess19.run(None, {"input": dummy_input})[0]
 
-    assert np.allclose(output21, output19, rtol=1e-5, atol=1e-6)
+    assert np.allclose(output21, output19, rtol=1e-3, atol=1e-6)
```

Comment:
Use differ rtol for onnxruntime < 1.21.1
Or xfail for this case

Reply:
Done